### ISSUE TITLE
Maximum parity size benchmark in distributed minio

### ISSUE THREAD

**Issue by deekoder** [minio/minio#4046]
Please explore this.

**Comment by deekoder** [comment]
- We have to test this first and then decide if we need to add support for this in Minio.
- This will also inform what we need to do on #4153

**Comment by fwessels** [comment]
Need to discuss results with AB and then plan accordingly. Depending on his schedule hopefully can do this either Monday or Tuesday.

**Comment by deekoder** [comment]
Fixed 16. Marked unblocked.

**Comment by deekoder** [comment]
Move this to prototype milestone.

**Comment by fwessels** [comment]
We did some benchmark testing with higher levels of data + parity combinations. Although it works perfectly fine, at higher levels such as 64 or 96 of data + parity the overall performance becomes primarily related to the file creation of many (small) files for average size objects (eg 1 MB per object).

As such we decided to keep the maximum at 16 for now and instead investigate scaling of buckets across multiple sets of drives (see #4153).
